### ISSUE TITLE
Null Reference Exception in ServerManager.cs

### ISSUE THREAD

**Issue by jimdroberts** [FirstGearGames/FishNet#484]
**General**
Fish-Networking version: Current

**Description**
Line 705 in ServerManager.cs throws a null reference exception for conn.ClientId if TryGetValueIL2CPP somehow fails to get the NetworkConnection.

**Expected behavior**
Change {conn.ClientId} to {args.ConnectionId} to fix the null reference exception and disconnect the correct NetworkConnection.
The line should be:
Kick(args.ConnectionId, KickReason.UnexpectedProblem, LoggingType.Error, $"ConnectionId {args.ConnectionId} not found within Clients. Connection will be kicked immediately.");

**Comment by FirstGearGames** [comment]
Yep! We caught this actually the other day and it's been fixed in 3.11.3.
